applications/StructuralMechanicsApplication/custom_elements/cr_beam_element_3D2N.hpp

```diff
@@ -26,10 +26,17 @@
 
 namespace Kratos
 {
+	/** 
+     * @class CrBeamElement3D2N
+     * 
+     * @brief This is a 3D-2node beam element with 3 translational dofs and 3 rotational dof per node
+     * 
+     * @author Klaus B Sautter
+     */
 
 	class CrBeamElement3D2N : public Element
 	{
-	private:
+	protected:
 		//const values
 		static constexpr int msNumberOfNodes = 2;

```

Comment:
this is OK,
     just as a comment, you could get it as
     const int Dim = GetGeometty().WorkingSpaceDimension()